doc/scratch-space.md

```diff
@@ -3,9 +3,8 @@ Containerized Data Importer(CDI) requires scratch space for certain operations t
 
 CDI uses the following mechanism to determine which storage class to use:
 
-1. Read the CDI config field _scratchSpaceStorageClass_ if that field exists, and the value matches one of the storage classes in the cluster, it will be used to create scratch space.
-2. If 1 is empty or didn't match a storage class, look up the _default_ storage class in the cluster, if that exists use that storage class.
-3. If there is no default storage class in the cluster, then use the storage class of the PersistentVolumeClaim(PVC) that is backing the DV that started the CDI operation.
+1. Read the CDI config field _scratchSpaceStorageClass_ if that field exists, and the value matches one of the storage classes in the cluster, it will be used to create scratch space. (This field could be set in manual or by fetching _default_ storage class in the cluster)
```

Comment:
Note the manually change.
```
This field could be set manually or by fetching _default_ storage class in the cluster
```

Reply:
Thanks for the review !! I applied your comments.